src/sick_tim_common.cpp

```diff
@@ -86,7 +86,6 @@ int SickTimCommon::stop_scanner()
 SickTimCommon::~SickTimCommon()
 {
   delete diagnosticPub_;
-  delete parser_;
```

Comment:
@mintar Yeah I was a little surprised this was freed not where it was declared.

I would have left it but it turns out that freeing the parser resources caused a segfault when you try to use the parser again.  :)